resolwe_bio/descriptors/unified-descriptor.yml

```diff
@@ -60,1404 +60,1412 @@
-              required: false
-            - name: stromal_score
-              label: Stromal score
-              type: basic:decimal
-              description: |
-                Stromal classification of tumors by quantification of stromal cells in TME.
-              required: false
-            - name: emt_score
-              label: EMT score
-              type: basic:decimal
-              description: |
-                Epithelial–mesenchymal transition score.
-              required: false
-            - name: msi_associated_gene_expressions
-              label: MSI associated gene expressions
-              type: basic:decimal
-              required: false
-            - name: description_of_signatures
-              label: Description of signatures
-              type: basic:string
-              required: false
-        - name: public_or_private_database_data
-          label: Public or private database data
-          group:
-            - name: database_1
-              label: Database 1
-              type: basic:string
-              required: false
-            - name: database_2
-              label: Database 2
-              type: basic:string
-              required: false
-            - name: database_3
-              label: Database 3
-              type: basic:string
-              required: false
-            - name: database_4
-              label: Database 4
-              type: basic:string
-              required: false
+        - name: patient_information__sample_label
+          label: Patient information/Sample label
+          type: basic:string
+          required: false
+        - name: patient_information__patient_id
+          label: Patient information/Patient ID
+          type: basic:string
+          required: false
+        - name: patient_information__batch
+          label: Patient information/Batch
+          type: basic:integer
+          required: false
+        - name: patient_information__group
+          label: Patient information/Group
+          type: basic:string
+          required: false
+        - name: patient_information__sex
+          label: Patient information/Sex
+          type: basic:string
+          required: false
+          choices:
+            - label: M
+              value: m
+            - label: F
+              value: f
+        - name: patient_information__race
+          label: Patient information/Race
+          type: basic:string
+          required: false
+          choices:
+            - label: White
+              value: white
+            - label: Black or African American
+              value: black or african american
+            - label: American Indian or Alaska Native
+              value: american indian or alaska native
+            - label: Asian
+              value: asian
+            - label: Native Hawaian or Pacific Islander
+              value: native hawaian or pacific islander
```

Comment:
Should be Hawaiian with double i.

Reply:
Different sources also use different spelling ([see](https://ontobee.org/ontology/NCIT?iri=http://purl.obolibrary.org/obo/NCIT_C17049#:~:text=Unknown%3B%20Multiracial%3B%20White%3B-,Native%20Hawaiian%20or%20Other%20Pacific%20Islander,-%3B%20Black%20or%20African)).